i18n/en.json

```diff
@@ -207,7 +207,7 @@
     "oauth_storage_quota_claim": "Storage quota claim",
     "oauth_storage_quota_claim_description": "Automatically set the user's storage quota to the value of this claim.",
     "oauth_storage_quota_default": "Default storage quota (GiB)",
-    "oauth_storage_quota_default_description": "Quota in GiB to be used when no claim is provided (Enter 0 for unlimited quota).",
+    "oauth_storage_quota_default_description": "Quota in GiB to be used when no claim is provided.",
```

Comment:
You should create a new key for this, since otherwise this won't show up as "untranslated" in other languages.